rules/xcodeproj.bzl

```diff
@@ -589,6 +592,13 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
         for arg in commandline_args_tuple:
             scheme_action_details["commandLineArguments"][arg] = True
 
+        # From https://developer.apple.com/documentation/xcode-release-notes/xcode-12-release-notes
+        #
+        # You can now specify a path for the LLDB init file to use in a Run and Test action.
+        # Configure this path in the Info tab of a scheme’s Run or Test action.
+        # The path can contain a build settings macro such as ${SRCROOT}, so the file can be part of a project. (38677796) (FB5425738)
+        scheme_action_details["customLLDBInit"] = lldbinit_file
```

Comment:
should we be explicit that this only supports xcode 12 and above? aka if one uses xcode 11, all these settings won't take effect?

Reply:
it seems like we are intentionally dropping xcode11 support .. I am OK with that, but suggested Thiago update the README accordingly.